packages/pigeon/lib/generator.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'ast.dart';
+
+/// A Super class of generator classes.
+///
+/// This is meant to provide structure and direction for future generator work.
```

Comment:
We probably won't remember to change this later; better to describe the intended structure rather than future work. E.g., "This provides the structure that is common across generators for different languages."